beta/src/components/MDX/Heading.tsx

```diff
@@ -27,11 +27,11 @@ const Heading = forwardRefWithAs<HeadingProps, 'div'>(function Heading(
         {isPageAnchor && (
           <a
             href={`#${id}`}
-            className={anchorClassName}
-            aria-hidden={true}
-            style={{
-              display: Comp === 'h1' ? 'none' : 'inline-block',
-            }}>
+            className={cn(
+              anchorClassName,
+              Comp === 'h1' ? 'hidden' : 'inline-block'
+            )}
+            aria-hidden={true}>
```

Comment:
Should this be a string or Boolean?

Reply:
This should be a string. I don't remember making any change here (it should have been boolean already).
https://github.com/reactjs/reactjs.org/blob/40c8cd2df94507b2a95f3d3c3d2310034556a602/beta/src/components/MDX/Heading.tsx#L31